Libplanet.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33424.131
```

Comment:
It probably doesn't matter much, but maybe we should keep this intact for now and have only necessary changes to the `sln` file. 🤔

Reply:
Applied by amending 9516a107b9a91c08e749ce967d01d45d6ea66b84.